css-transitions/styles.css

```diff
@@ -0,0 +1,54 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+  background-color: rgb(245 245 245);
+}
+
+section {
+  max-width: 992px;
+  margin: 2rem auto;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.custom-button {
+  padding: 0.5rem 1.5rem;
+  border: none;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+  color: white;
+  background-color: blue;
+}
+
+.custom-button:hover {
+  transition: 0.5s;
+  background-color: red;
+}
+
+.card-wrapper {
+  text-align: center;
+}
+
+.card {
+  max-width: 18rem;
+  padding: 0.5rem;
+  margin-bottom: 2rem;
+  border: 1px solid rgb(228 228 228);
+  border-radius: 0.25rem;
+  background-color: white;
+  cursor: pointer;
+  transform: scale(1);
+}
+
+.card:hover {
+  transition: 0.5s;
```

Comment:
make sure that the transition property is on the main css class, not the one with the pseudoclass. That will ensure that the transition works for both the addition of the property, and the removal of it.